core/pom.xml

```diff
@@ -178,6 +190,11 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-oauth2-client</artifactId>
```

Comment:
jfrog is flagging these spring dependencies  as high severity
https://github.com/pentaho/pentaho-platform/pull/5879#issuecomment-2828299126

can you speak to these?

Reply:
@NJtwentyone I believe this work is being handled by the Rover team on upgrade tomcat and spring. @hv-vamsi  can you please confirm that these issues will be addressed by the spring upgrade ?

Reply:
Yes. It will be taken care as part of Tomcat-10.X upgrade. Once this code is merged to master, will rebase this commit to Tomcat-10.X branch and make necessary changes. Checked with Luc as well on the same and he okayed it.

Already, tested as sanity check, need to update some of the imports and then it just worked as expected. No changes to the source code.